docs.helm.sh/gulpfile.js

```diff
@@ -243,8 +244,12 @@ gulp.task('clone', function() {
 
   gulp.task('redirect-anchor', function() {
     return gulp.src('source/docs/**/*.md')
-      // update installation guides
+      // update quickstart and install links
       .pipe(replace(/\]\(.*install\.md/, '](../using_helm/#installing-helm'))
+      .pipe(replace('#Install-Helm', '#installing-helm'))
+      .pipe(replace('#quickstart]', '#quickstart-guide]'))
```

Comment:
out of curiosity, are these closing brackets intentional?

Reply:
Yeah, I want to replace a specific instance of #quickstart in a markdown url, without impacting on any correct links which would otherwise get rewritten and broken as follows:

```
without limiting with a closing ']'

#quickstart         ->   #quickstart-guide
#quickstart-guide   ->   #quickstart-guide-guide
```